### DISCUSSION TITLE
[Feature] Warning users prior to a possible data loss

### DISCUSSION THREAD

**Discussion by Juan-Esc** [immich-app/immich#14000] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

**TL;DR This feature consists on warning users prior to a possible data loss due to server admin not being able to maintain and/or pay for the server.**

**Introduction:**

**Feature:**

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is not something for Immich to handle.
